discordbot/merriam-webster/client.go

```diff
@@ -24,26 +24,33 @@ func init() {
 // Client describes access to the Merriam-Webster dictionary at dictionaryapi.com
 type Client interface {
 	SearchCollegiate(word string) ([]types.CollegiateResult, []string, error)
+	IsConfigredCorrectly() bool
```

Comment:
Kind of a shame to be storing the init error without doing anything with it. Why not make this `GetInitializationError() error` instead, and return the error? Then it can be checked as `err == nil` or `err != nil`, and its string message is `err.Error()`. That string can then be printed in `SearchCollegiate`, giving a *reason* why it failed.